src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,252 @@
+					.setDescription(message.language.tget('COMMAND_HIGHERLOWER_EMBED_DESCRIPTION', game.number))
+					.setFooter(message.language.tget('COMMAND_HIGHERLOWER_EMBED_FOOTER'))
+			});
+
+			// Add the options
+			const filterFunction = (e: ReactionEmoji, t: number) => e !== ReactionEmoji.OK && ((t === 1 && e !== ReactionEmoji.CASHOUT) || (t !== 1 && e !== ReactionEmoji.CANCEL));
+			const reactArray = Object.values(ReactionEmoji).filter(e => filterFunction(e, game.turn));
+			for (const emoji of reactArray) {
+				await game.gameMessage.react(emoji);
+			}
+
+			// Grab the user selection
+			const emojiSelected = await new Promise<ReactionEmoji>(resolve => {
+				// Create the timeout
+				const timeout = this.client.setTimeout(async () => {
+					await this.end(game, message, game.turn > 1);
+				}, 3 * Time.Minute);
+
+				// Insert weird promise tomfoolery. I still havent completely understood how this works. Something something weird promise resolves
+				emojiPromiseContext = {
+					resolve,
+					timeout
+				};
+			});
+
+			// Main game logic (at last)
+			const oldNum = game.number;
+			game.number = this.random(game.number);
+
+			switch (emojiSelected) {
+				case ReactionEmoji.HIGHER:
+					game.running = await (oldNum < game.number ? this.win(game, message) : this.loss(game, message));
+					break;
+
+				case ReactionEmoji.LOWER:
+					game.running = await (oldNum > game.number ? this.win(game, message) : this.loss(game, message));
+					break;
+
+				case ReactionEmoji.CASHOUT:
+				case ReactionEmoji.CANCEL:
+					await this.end(game, message, emojiSelected === ReactionEmoji.CASHOUT);
+					game.running = false;
+					break;
+			}
+
+			if (game.running) game.turn++;
+		}
+	}
+
+	private async win(game: HigherLowerGameData, message: KlasaMessage) {
+		const { author: user, language } = message;
+		await game.gameMessage.reactions.removeAll();
+		// TODO(Quantum): Winning event
+		await game.gameMessage.edit({
+			content: '',
+			embed: new MessageEmbed()
+				.setColor(Slotmachine.SUCCESS_COLOUR)
+				.setTitle(language.tget('COMMAND_COINFLIP_WIN_TITLE'))
+				.setDescription(language.tget('COMMAND_HIGHERLOWER_WIN_DESCRIPTION', this.calculateWinnings(game.wager, game.turn), game.number))
+				.setFooter('Act fast! You don\'t have much time')
+		});
+		const reactArray = Object.values(ReactionEmoji).filter(e => e !== ReactionEmoji.HIGHER && e !== ReactionEmoji.LOWER && e !== ReactionEmoji.CASHOUT);
+		for (const emoji of reactArray) {
+			await game.gameMessage.react(emoji);
+		}
+		const reactionMap = await game.gameMessage.awaitReactions((r, u) => reactArray.includes(r.emoji.name) && u.id === user.id, { max: 1, time: 10000 });
+		await game.gameMessage.reactions.removeAll();
+
+		const whatToDo = reactionMap.size === 0
+			? EndingAction.TIMEOUT
+			: reactionMap.first()!.emoji.name === ReactionEmoji.OK
+				? EndingAction.PLAY
+				: EndingAction.STOP;
```

Comment:
```suggestion
				: EndingAction.STOP;
				
```

new line before the switch would be preferred